float32 percentage
---
int8 status
